src/Redux/Reducers/userSlice.js

```diff
@@ -0,0 +1,119 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+import { AlertToast, SuccessToast } from "../../components/toasts";
+const initialState = {
+  users: [],
+};
+
+export const getAllUsers = createAsyncThunk(
+  "users/getAllUsers",
+  async (mockParameter, { rejectWithValue }) => {
```

Comment:
why you passed mockParameter here?

Reply:
If we have to make a request without passing any data the params will be like this `(anyParameter ,{rejectWithValue})=>{}` in the case if u need to pass  `rejectWithValue`